utils/debug.c

```diff
@@ -104,7 +133,9 @@ static bool check_busybox(const char *pager)
 check:
 	path = realpath(pager, NULL);
 	if (path) {
-		ret = !strncmp("busybox", basename(path), 7);
+		if (!strncmp("busybox", basename(path), 7) ||
+		    !is_elf_file(path))
```

Comment:
I don't think this is a good guess.. if it's not an ELF binary, you should search it has some string like busybox.

Reply:
Do you mean `is_elf_file` is called first, then check whether it has a substring "busybox" rather than `strncmp` with `basename`?

Reply:
I mean that if it's a script, then it should search "busybox" in the shebang.